overlord/snapstate/autorefresh_gating_test.go

```diff
@@ -149,8 +163,19 @@ slots:
         interface: iface1
 `
 
+const coreYaml2 = `name: core
+type: os
+version: 1
+slots:
+    slot:
```

Comment:
shouldn't this be called desktop for clarity like the real one?

Reply:
Sure, will do.